src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -0,0 +1,65 @@
+import { ModularTable } from "@canonical/react-components";
+import { useMemo } from "react";
+import { useParams } from "react-router-dom";
+import type { Column } from "react-table";
+
+import type { EntityDetailsRoute } from "components/Routes/Routes";
+import { generateFakeAuditLogs } from "pages/Logs/audit-logs-fake-data";
+
+const AuditLogsTable = () => {
+  const { modelName } = useParams<EntityDetailsRoute>();
+
+  const emptyMsg = `There are no audit logs available yet for ${modelName}`;
+  const columnData: Column[] = useMemo(
+    () => [
+      {
+        Header: "user tag",
+        accessor: "userTag",
+      },
+      {
+        Header: "model",
+        accessor: "model",
+      },
+      {
+        Header: "time",
+        accessor: "time",
+      },
+      {
+        Header: "facade name",
+        accessor: "facadeName",
+      },
+      {
+        Header: "facade method",
+        accessor: "facadeMethod",
+      },
+      {
+        Header: "facade version",
+        accessor: "facadeVersion",
+      },
+    ],
+    []
+  );
+  const fakeTableData = useMemo(() => {
+    const auditLogsData = generateFakeAuditLogs();
+    const tableData = auditLogsData.map((auditLogsEntry) => {
+      const { model } = auditLogsEntry;
+      const time = new Date(auditLogsEntry.time).toLocaleString();
+      const userTag = auditLogsEntry["user-tag"];
```

Comment:
This should remove the `user-` at the start of the tag, to go from the tag to just the username.

Reply:
Makes sense! As `.replace("user-", ""0` is utilised in a lot of places, I've created the util function `getUserName` and replaced it where necessary.

Reply:
Thanks!